src/index.js

```diff
@@ -135,6 +141,7 @@ function parseDocument(doc) {
   // existing fragments of the same name
   parsed = processFragments(parsed);
   parsed = stripLoc(parsed, false);
+  parsed.toString = toString.bind(null, parsed.loc);
```

Comment:
This isn't guaranteed to work if the query AST is mutated, right? Is there something we can do to guard against that?

Reply:
I'll write a test case for it and fix.

Reply:
`graphql.print()` might be better:

```
import { print } from "graphql/language";

parsed.toString = function () { return print(this); };
```